arrow-fx-coroutines/src/main/kotlin/arrow/fx/coroutines/Duration.kt

```diff
@@ -1,19 +1,35 @@
+@file:Suppress("DEPRECATION_ERROR")
+
 package arrow.fx.coroutines
 
 import java.util.concurrent.TimeUnit
 
-// TODO replace with kotlin.time
+@Deprecated(
+  "Redundant since 0.12.0 will provide automatic integration with KotlinX",
```

Comment:
I think we need to provide a clearer message here. `Duration` is `Deprecated` in favor of `kotlin.time.Duration`, and KotlinX offers all APIs we offer in `timer.kt` for both `kotlin.time.Duration` and `milliseconds: Long` so APIs are available to the user with and without optining in for `kotlin.time`.

But that's a bit too long for a deprecated message I guess 😅

Reply:
Yes, I was also unsure, what message would make sense in this case. 
But I will go for yours, regardless 😄